### PR TITLE
docs: simplifying calling other APIs and Web services

### LINKED ISSUE #6369: In consistent documentation for Calling-other-APIs-and-web-services
In the documentation for https://loopback.io/doc/en/lb4/Calling-other-APIs-and-web-services.html

The example code is not consistent, it switches between a calculator with 2 parameters in the service, and using 1 parameter in datasourse example.

It also does not show how to pass different types of parameters, like an object from the calling code through to the final API.

Showing how to pass an instantiated MODEL would be very useful, and it should be shown as an example.

## Acceptance Criteria
- [ ] split the instructions to call SOAP and REST services as separate page
- [ ] in each instruction, include examples that involves other types in parameters.
- [ ] For calling REST service, include:
  - [ ] docs to explain the template if the [REST connector readme](https://loopback.io/doc/en/lb4/REST-connector.html) doesn't contain sufficient information
   - [ ] example to call POST (there's only GET in the example)

### PR DESCRIPTION
**Pull request by mrmodise**
Closes #6369

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by raymondfeng — APPROVED** [review]

File: docs/site/Accessing-services.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+Please refer to
+[Testing Your Application](./Testing-your-application.md#test-your-services-against-real-backends)
+for guidelines on integration testing.
+
+## Try it out
+
+- [Calling SOAP Web services](Calling-soap-web-services.md)
+- [Calling REST APIs services](Calling-rest-api-services.md)
```
Comment (dhmlau) [inline docs/site/Accessing-services.md:75]: To match with what's in `Calling-rest-apis.md` in your PR: 

```suggestion
- [Calling REST APIs](Calling-rest-apis.md)
```

File: docs/site/Calling-rest-apis.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
+---
+lang: en
+title: 'Calling REST APIs'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, REST, SOAP
```
Comment (dhmlau) [inline docs/site/Calling-rest-apis.md:4]: Remove `SOAP` in the keyword list? 

```suggestion
keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, REST
```

File: docs/site/Calling-rest-apis.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Calling-rest-apis.html
+---
+
+To get started with accessing REST APIs you need to create a datasource, as
```
Comment (dhmlau) [inline docs/site/Calling-rest-apis.md:9]:
```suggestion
To get started with accessing REST APIs, you need to create a datasource as
```

File: docs/site/Calling-soap-web-services.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
+---
+lang: en
+title: 'Calling SOAP Web services'
```
Comment (dhmlau) [inline docs/site/Calling-soap-web-services.md:3]: Making the title to have the first character of each word to be upper case (consistent with other docs).
```suggestion
title: 'Calling SOAP Web Services'
```

File: docs/site/Calling-soap-web-services.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, REST, SOAP
```
Comment (dhmlau) [inline docs/site/Calling-soap-web-services.md:4]: Similarly, maybe remove `REST`? 
```suggestion
keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, SOAP
```

File: docs/site/Calling-soap-web-services.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Calling-soap-web-services.html
+---
+
+To get started with accessing SOAP Web services you need to create a datasource,
```
Comment (dhmlau) [inline docs/site/Calling-soap-web-services.md:9]: Similar suggestion as in the "calling REST APIs" page:
```suggestion
To get started with accessing SOAP Web services, you need to create a datasource
```

**Review by dhmlau — COMMENTED** [review]
@mrmodise, thanks for splitting up the how-tos for calling REST vs SOAP services. I have some minor comments.

File: docs/site/Calling-rest-apis.md (hunk 3 above)
Comment (mrmodise) [reply docs/site/Calling-rest-apis.md:9]: Done

File: docs/site/Calling-soap-web-services.md (hunk 5 above)
Comment (mrmodise) [reply docs/site/Calling-soap-web-services.md:4]: Done

File: docs/site/Calling-soap-web-services.md (hunk 6 above)
Comment (mrmodise) [reply docs/site/Calling-soap-web-services.md:9]: Done

File: docs/site/Calling-soap-web-services.md (hunk 4 above)
Comment (mrmodise) [reply docs/site/Calling-soap-web-services.md:3]: Done

File: docs/site/Calling-rest-apis.md (hunk 2 above)
Comment (mrmodise) [reply docs/site/Calling-rest-apis.md:4]: Done

File: docs/site/Accessing-services.md (hunk 1 above)
Comment (mrmodise) [reply docs/site/Accessing-services.md:75]: Done

**Review by dhmlau — APPROVED** [review]

**Comment by mrmodise** [comment]
@raymondfeng @dhmlau looks like Travis CI build is hanging forever

**Comment by mrmodise** [comment]
@bajtos looks like its waiting on your rubber stamp 😄

**Comment by dhmlau** [comment]
@mrmodise, we were having some issue with Travis, so the build was pending. The PR should be good to go. Thanks for the ping.

**Pull request merged**